systems/data-plan/package/pkg/db/package_repo.go

```diff
@@ -40,22 +46,21 @@ func (p *packageRepo) Get(orgId, id uint64) ([]Package, error) {
 	return packages, nil
 }
 
-func (r *packageRepo) Delete(orgId , packageId uint64, nestedFunc ...func() error) error {
-	err := r.Db.ExecuteInTransaction(func(tx *gorm.DB) *gorm.DB {
-		d := tx.Delete(&Package{}, "id = ? AND org_id = ?", packageId, orgId)
+func (r *packageRepo) Delete(orgId uint64, packageId uint64) error {
+	err := r.Db.GetGormDb().Transaction(func(tx *gorm.DB) error {
```

Comment:
Again, we can skip the transaction and  go directly to the `Where` clause.

Reply:
addressed

Reply:
We are still using a transaction here.  It is not necessary here, but  we can keep it if you like.